toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -291,6 +294,15 @@ func CustomizeImage(buildDir string, baseConfigPath string, config *imagecustomi
 		return fmt.Errorf("failed to customize raw image:\n%w", err)
 	}
 
+	if config.Output.Artifacts != nil && config.Output.Artifacts.Path != "" {
```

Comment:
The `config.Output.Artifacts.Path != ""` check is redundant, due to the API validity checks.